src/MasterSite.php

```diff
@@ -195,6 +195,18 @@ function ( $block_content, $block ) {
 		add_action( 'admin_notices', [ $this, 'show_dashboard_notice' ] );
 		add_action( 'wp_ajax_dismiss_dashboard_notice', [ $this, 'dismiss_dashboard_notice' ] );
 		add_filter( 'timber/twig', [ $this, 'p4_optimize_img_url' ] );
+
+		// Disable WordPress(WP5.5) Block Directory.
+		global $wp_version;
+		if ( version_compare( $wp_version, '5.5', '>=' ) ) {
```

Comment:
Just to avoid the check, I'm thinking that we could create a new `wp55` branch out of master and merge this there. Because I think we may have other similar code changes that need to happen before we upgrade to 5.5

Reply:
The `wp55` branch is already exist, but far behind of `master`. On merging of this PR, all `master` branch changes will automatically add in `wp55` branch.

Reply:
Probably created that branch a long time :) I just rebased it and commits got cleared.